app/javascript/packages/document-capture/document-capture.scss

```diff
@@ -1,4 +1,5 @@
-@import 'identity-style-guide/dist/assets/scss/packages/required';
+@import '@18f/identity-idp-required-scss';
+@import '@18f/identity-components/troubleshooting-options';
```

Comment:
It feels a little fragile to have to pull in stylesheets based on the components being used in this pack. Also, this is duplicating the styles into the `document-capture` pack, rather than using the existing pack. I might want to think on this a bit more...